packages/maven.rb

```diff
@@ -3,11 +3,11 @@
 class Maven < Package
   description 'Apache Maven is a software project management and comprehension tool.'
   homepage 'https://maven.apache.org/'
-  version '3.9.0'
+  version '3.9.7'
   license 'Apache-2.0'
   compatibility 'all'
-  source_url 'https://dlcdn.apache.org/maven/maven-3/3.9.0/binaries/apache-maven-3.9.0-bin.tar.gz'
-  source_sha256 'b118e624ec6f7abd8fc49e6cb23f134dbbab1119d88718fc09d798d33756dd72'
+  source_url 'https://dlcdn.apache.org/maven/maven-3/3.9.7/binaries/apache-maven-3.9.7-bin.tar.gz'
+  source_sha256 'c8fb9f620e5814588c2241142bbd9827a08e3cb415f7aa437f2ed44a3eeab62c'
 
   depends_on 'openjdk8' unless File.exist? "#{CREW_PREFIX}/bin/java"
```

Comment:
We can probably just have `depends_on 'openjdk8'`, right?

Reply:
Well, it works with any version of java so...